### PR TITLE
disqus_config to avoid duplicate/split comments

### PR DESCRIPTION
**Pull request by spradnyesh**
as explained in
- https://help.disqus.com/customer/en/portal/articles/472097
- https://help.disqus.com/customer/en/portal/articles/2158629

### CODE REVIEW

File: src/leiningen/new/cryogen/themes/blue/html/post.html [hunk 1]
```diff
@@ -15,6 +15,10 @@
     {% if disqus-shortname %}
     <div id="disqus_thread"></div>
     <script type="text/javascript">
+        var disqus_config = function () {
+              this.page.url = {{post.uri}};
```
Comment (yogthos) [inline src/leiningen/new/cryogen/themes/blue/html/post.html]: I think you need quotes around the values, e.g: `this.page.url = "{{post.url}}";

**Comment by spradnyesh** [comment]
gentle reminder :)

**Comment by lacarmen** [comment]
Sorry @spradnyesh! I didn't realize you'd pushed another commit

**Comment by spradnyesh** [comment]
np. happens :)

warm regards,
Pradnyesh Sawant
--
That's a pretty clear strategy: Investigate, Design, Experiment, Adjust.
What a great I.D.E.A. --http://www.openforum.com/idea-hub/topics/the-
world/article/design-thinking-101-matthew-e-may

**Comment by spradnyesh** [comment]
please do *not* pull the PR; there's a bug i'm trying to fix. will submit another PR shortly :)

**Comment by spradnyesh** [comment]
submitted the updated PR

using "{{site-url}}{{uri}}" instead of "{{page.uri}}"

tested that it works on my setup

**Comment by lacarmen** [comment]
The url looks correct now but when I tried the change on my personal blog and it caused Disqus to be unable to load. Do you know what's going on here?

> screen shot 2016-08-17 at 10 40 23 pm

**Comment by lacarmen** [comment]
Oh I see, it doesn't load properly if you're running your blog locally. 

LGTM then!

**Comment by spradnyesh** [comment]
for running locally, set the :site-url in config.edn to "http://localhost/"

warm regards,
Pradnyesh Sawant
--
Half the money I spend on advertising is wasted; the trouble is I don't
know which half. --John Wanamaker

**Comment by spradnyesh** [comment]
thanks :)

warm regards,
Pradnyesh Sawant
--
Half the money I spend on advertising is wasted; the trouble is I don't
know which half. --John Wanamaker

**Pull request merged**